internal/pkg/cli/pipeline_delete.go

```diff
@@ -144,6 +156,16 @@ func (o *deletePipelineOpts) Execute() error {
 	return nil
 }
 
+func askDeployedPipelineName(sel codePipelineSelector, appName, msg string) (string, error) {
+	pipeline, err := sel.DeployedPipeline(fmt.Sprintf(msg, color.HighlightUserInput(appName)), "", map[string]string{
```

Comment:
Instead of taking the fmtString, can we let the client generate it:
```suggestion
func askDeployedPipelineName(sel codePipelineSelector, appName, msg string) (string, error) {
	pipeline, err := sel.DeployedPipeline(msg, "", map[string]string{
```